cachier/core.py

```diff
@@ -220,12 +173,17 @@ def cachier(
         warn(message, DeprecationWarning, stacklevel=2)
         hash_func = hash_params
     # Override the backend parameter if a mongetter is provided.
```

Comment:
This comment should be at line 184 after the edits, as it describes the `if callabe(mongetter):` logic.